README.md

```diff
@@ -1,2 +1,154 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project is implemented using the Ruby programming language, known for its elegant syntax and flexibility. Ruby provides a suitable foundation for this Morse code decoding project due to its ease of use and string manipulation capabilities.
+
+## Key Features <a name="about-project"></a>
+- [x] Implementation of functions using `def ...end` syntax.
+- [x] Display output using `puts ...`.
+- [x] Create classes using `class ...end` syntax.
+
+## Future Features <a name="future-project"></a>
+- Command-line Interface: Enhance the app by adding a command-line interface for easy interaction and management of the library's functions.
+- Book Recommendations: Implement a feature that suggests related books to borrowers based on their borrowing history and interests.
+- Notifications System: Develop a notifications system to remind borrowers of upcoming due dates for returned books.
+
+## Getting Started <a name="getting-started"></a>
+1. Clone the project repository:
+   ```
+   git clone https://github.com/Zven94/OOP-school-library.git
+   ```
+## Setup <a name="setup"></a>
+To set up the project locally, follow these steps:
+1. Clone the project repository:
+   ```
+   git clone https://github.com/Zven94/OOP-school-library.git
+   ```
+2. Navigate to the project folder:
+   ```
+   cd school_library
+   ```
+
+### Prerequisites <a name="prerequisites"></a>
+- An integrated development environment (IDE) like Visual Studio Code.
+- Ruby installed on your machine.
+- Git installed on your machine.
+- A GitHub account for version control.
+
+# Usage of Ruby <a name="usage"></a>
+
+Follow these steps to use the program:
+
+1. Open your terminal.
+2. Navigate to the directory where the program files are located using the `cd` command.
+
+```bash
+cd /path/to/your/program/files
+```
+
+3. Start the Interactive Ruby (IRB) console by typing `irb` and pressing Enter.
+
+```bash
+irb
+```
+
+4. Inside the IRB console, load the `test.rb` file to execute the program.
+
+```ruby
+load 'test.rb'
+```
+
+5. The program will run, and you will see the output in the console.
+
+```
+maximilianus
+Maximilianus
+Maximilian
+Classroom label: Math
+Student 1: Alice (16 years old)
+Student 2: Bob (17 years old)
+Rental date: 2023-08-30
+Rental person: John
+Rental book: The Great Gatsby
+Person's rentals: ["The Great Gatsby"]
+Book's rentals: ["John"]              
+=> true
+```
+
+That's it! You've successfully executed the program using IRB.
+
+Remember to replace `/path/to/your/program/files` with the actual path to the directory where your program files are located.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+👤 **Nicolas Bolañez**
+- GitHub: [@Nico](https://github.com/Zven9)
+- LinkedIn: [@Nico](https://www.linkedin.com/in/nico-emiliano/)
+
```

Comment:
- [ ] I would suggest you please provide the correct link to you Github page and LinkedIn as it says 404 error.